src/content/platform-overview/architecture/index.md

```diff
@@ -0,0 +1,156 @@
+
+As explained previously, both the management cluster and the workload cluster(s) have the same structure and configuration. Giant Swarm also collaborates on [Cluster API](https://cluster-api.sigs.k8s.io/) to simplify the bootstrap and configuration of the cluster infrastructure and all the components needed for a cluster to function.
+
+![Cluster architecture image](CAPI_architecture.png)
+
+By default, the machines are spread across three different failure domains, or zones, to ensure the availability of the API and workloads running on top. In our setup, three control plane machines hold the Kubernetes API and the other controllers, and a variable number of worker machines contain the workloads.
+
+Besides the Kubernetes machines, we run a bastion host that helps us with the operations. It is the single entry point to the running infrastructure. That way all cluster machines can live in a private network and reduce the exposure of the services running on them via explicit configuration. We are working on a [Teleport](https://goteleport.com/) solution to replace the necessity of the bastions and centralize cluster access through a single tool.
+
+#### Providers
+
+Although we try to maintain feature parity between infrastructure providers, offering the same features and using the same configuration, there are some subtle differences depending on the underlying provider functionality.
+
+{{< tabs >}}
+{{< tab id="flags-clouddirector" title="VMware Cloud Director">}}
+
+*Compatibility*
+
+The setup supports organization virtual datacenters (OVCDs) running on VMware Cloud Director 10.3 and above. It must be backed by backed with NSX-T and NSX advanced load balancer (ALB) with the load balancer feature enabled on the Edge gateway. 
+
+*Authentication*
+
+Cluster API Provider VMware Cloud Director (CAPVCD), along with the associated Cloud Provider interface (CPI) and Container Storage interface (CSI), authenticate against the VMware Cloud Director API using an API Token (sometimes also referred to as Refresh Token) which is stored in a secret. Such a token can be created by any user with the right permission and can be revoked at any time should there be suspicion of it being compromised.
+
+*Networking*
+
+The kubernetes API and services of type `LoadBalancer` get IPs from a pool of external IPs available in the edge gateway. It can be set statically or takes the next available IP if unspecified. A virtual service is then created with the required IP/port and is associated with a load balancer pool that contains the relevant node IPs as members. For the CPI, we support the virtual service shared feature which was introduced in VCD 10.4 as well as the legacy method based on a single internal IP and multiple DNAT rules.
```

Comment:
```suggestion
The Kubernetes API and services of type `LoadBalancer` get IPs from a pool of external IPs available in the edge gateway. It can be set statically or takes the next available IP if unspecified. A virtual service is then created with the required IP/port and is associated with a load balancer pool that contains the relevant node IPs as members. For the CPI, we support the virtual service shared feature which was introduced in VCD 10.4 as well as the legacy method based on a single internal IP and multiple DNAT rules.
```